src/dvr/dvr_config.c

```diff
@@ -937,84 +937,84 @@ const idclass_t dvr_config_class = {
     {
       .type     = PT_BOOL,
       .id       = "day-dir",
-      .name     = N_("Make Subdirectories Per Day"),
+      .name     = N_("Make subdirectories per day"),
```

Comment:
subdirectory vs sub-directory... so long as it's consistent, I don't think it matters